feat: support pip_parse in Gazelle (#633)
bazelbuild/rules_python: gazelle/README.md

    modules_mapping = ":modules_map",
    # This is what we called our `pip_install` rule, where third-party
    # python libraries are loaded in BUILD files.
    pip_repository_name = "pip",

>     # If using pip_parse incremental, switch the following to True,

Reviewer: nit: users don't think of that name, they think of "pip_parse" - should probably say both

>     # If using pip_parse incremental, switch the following to True,

Author: ```suggestion
    # When using pip_parse instead of pip_install, set the following.
```

>     pip_repository_incremental = False,

Author: ```suggestion
    # pip_repository_incremental = True,
```
